src/coreComponents/physicsSolvers/solidMechanics/integratedTests/Sneddon.xml

```diff
@@ -126,8 +126,9 @@
       cellBlocks="{ cb1 }"
       materialList="{ rock }"/>
 
-    <EmbeddedSurfaceElementRegion
+    <SurfaceElementRegion
       name="Fracture"
+      subRegionType ="embeddedElement"
```

Comment:
@rrsettgast @af1990 @joshua-white before changing all the xml files I wanted to check with you that you like it this way. I have basically unified the two regions since they were almost the same and then added a field to choose the subRegionType.

Reply:
For what it can matter ... I like it.

Reply:
I like the merge as well.  Is it possible for the `subRegionType` to be determined when the mesh is generated, rather than by the user here?  In CellElements, for example, the user doesn't have to specify whether a sub-region is a hex or a tet, because this is obvious from the geometry.

Reply:
Put another way, the subRegionType should be known from the fact it will be created by the EmbeddedSurfaceGenerator.

Reply:
That's for sure my long term goal but I don't think we are there yet. The way fractures are defined in the two cases are very different. Unless we only register the subRegion on the region in the surfaceGenerator itself. I don't like this solution too much though coz I feel it's not really its responsibility.

Reply:
I will think about it though and maybe do some tests.

Reply:
I think we can just have a subregion for each type...and fill them accordingly. I don't think it is a stretch from what you already have?

Reply:
I can register both `subRegions`. One would simply be empty... At the end of the day even for the background mesh the user does need to specify the `elementType`.